lib/archethic/db/embedded_impl/chain_index.ex

```diff
@@ -93,11 +102,15 @@ defmodule Archethic.DB.EmbeddedImpl.ChainIndex do
       # Register last addresses of genesis address
       {last_address, timestamp} = get_last_chain_address(genesis_address, db_path)
 
+      # Write fast lookup entry for this transaction on memory
       true =
         :ets.insert(
           @archethic_db_last_index,
           {genesis_address, last_address, DateTime.to_unix(timestamp, :millisecond)}
         )
+
+      ## We trigger the loading of the last address for this chain in the LRU cache
+      _ = search_tx_entry(last_address, db_path)
```

Comment:
Why do you need to trigger the loading ? I think this is done in the `do_scan_summary_table`fn.

Reply:
I am doing to be sure the last index of each chains would be inserted.
Julien advised to keep filling the cache in do_scan_summary_table/1
Then  I thought it would be good to indeed keep the last transaction indexes, but conceptually previous step doesn't guaranty this as some entry might be evicted
As search entry put found results into cache, I am triggering it with the last transaction address
But it is true I could remove this seach, and rely on the fact the cache will rarely be full... hopefully :)

Reply:
this is because we are loading the from the summary and not the chain's file .
but at the of `do_scan_summary_table/1` the latest transaction would have been prioritized.